### ISSUE TITLE
player and GM contributing to 

### ISSUE THREAD

**Issue by gtludwig** [crnormand/gurps#252]
Let's say there is a player who, while, gaming (or in his non-game time) decides to contribute to the internationalization drive. But this player does not know how to code or found a translation that is not properly accurate.

What if this player could click on a button on the window that is open. This button would highlight all the placeholders - everything that has {{localize "GURPS.*"}}. Then he would click on one of the place holders, which would pop up a box with that placeholder and the list of i18n terms already existent for that placeholder.

The player would be able to add a suggestion that would and click on a submit button.

This would appear at his GM's screen to add or edit the lang json.

And it would also be sent to us to evaluate and add to the oficial i18n file.

**Comment by crnormand** [comment]
Yeah, that could be really helpful.   We might be able to add behavior to the {{localize xxx}} handlebar handler to output some more css which we could look for.   When we (and by "we", I mean at has been all you, 100%) get done with internationalization, we should definitely investigate this.     I don't know how much we can automate... but if we can collect up the information at all, that would be a huge bonus.
